<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Pradeep_Machine" failures="2" tests="2" name="Test(failed)(failed)(failed)" time="42.501" errors="0" timestamp="2024-05-03T12:59:59 IST">
  <testcase classname="testCases.TC_002_SignupTestbyOTP" name="verify_signup" time="6.967">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_002_SignupTestbyOTP.verify_signup(TC_002_SignupTestbyOTP.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_signup -->
  <testcase classname="testCases.TC_001_SignupTestbyVerifyEmail" name="verify_signup" time="6.6">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_001_SignupTestbyVerifyEmail.verify_signup(TC_001_SignupTestbyVerifyEmail.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_signup -->
</testsuite> <!-- Test(failed)(failed)(failed) -->
